.github/workflows/release.yml

```diff
@@ -83,6 +83,9 @@ jobs:
     runs-on: ubuntu-latest
     environment: Publishing
     needs: [Create-NPM-Cache, Build]
+    permissions:
+      id-token: write
+      contents: read
```

Comment:
should we move permissions to top under [name](name: Release @skyscanner/backpack-web to NPM) as other jobs?